controllers/users.js

```diff
@@ -625,6 +625,31 @@ const updateRoles = async (req, res) => {
   }
 };
 
+const archiveUserIfNotInDiscord = async (req, res) => {
+  try {
+    const data = await userQuery.archiveUserIfNotInDiscord();
+
+    if (data.totalUsers === 0) {
+      return res.status(200).json({
+        message: "Couldn't find any users currently inactive in Discord but not archived.",
+        data,
+      });
+    }
+
+    if (data.totalOperationsFailed === data.totalUsers) {
+      return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
+    }
+
+    return res.status(200).json({
+      message: "Successfully updated users archived role to true if in_discord role is false",
+      data,
```

Comment:
Can you also send the updated user length with debug param?

Reply:
I am sending the updated user length in the data I have passed.

```
"data": {
        "totalUsers": "number",
        "totalUsersArchived": "number",
        "totalOperationsFailed": "number"
      }
```

Here total users archived denote the number of users updated.